src/kronecker.jl

```diff
@@ -119,9 +119,9 @@ end
     na, ma = size(At)
     mb, nb = size(B)
     if nb*ma < mb*na
-        _unsafe_mul!(reshape(y, (mb, ma)), B, convert(Matrix, X*At))
+        _unsafe_mul!(reshape(y, (mb, ma)), B, convert(AbstractMatrix, X*At))
```

Comment:
i wish that somewhere near line 130 there would be a comment explaining what is unsafe here.
perhaps it is that the size of `y` is not checked?  if so, maybe a brief comment on why that's ok,
e.g., it is checked elsewhere?

lots of subtle stuff i don't quite get here so i can't offer much commentary on it.  seems ok...

Reply:
Yes, the `_unsafe` is due to lack of any size checking, as that is done (i) in `mul!` and (ii) during construction of the specific`LinearMap` types. I can add a comment in the main file to explain the global structure.